app/segments/drill/[id]/submission/index.js

```diff
@@ -1,19 +1,42 @@
+import { useLocalSearchParams } from "expo-router";
 import { useState } from "react";
 import { PaperProvider } from "react-native-paper";
-import { AttemptData } from "~/testData";
+
+import Loading from "~/components/loading";
 import Input from "./input";
+import Result from "./result";
+
+import { useDrillInfo } from "../../../../../hooks/useDrillInfo";
```

Comment:
this should also use the `~/hooks/useDrillInfo` format right?

Reply:
yes, I think this was altered when I got the force push from frank but I just updated it